### ISSUE TITLE
[Carousel A11y] Buttons are not visible in high-contrast

### ISSUE THREAD

**Issue by JeanRoca** [microsoft/AdaptiveCards#7659]
https://microsoft.visualstudio.com/OS/_workitems/edit/39443457
